index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Travel Blog</title>
     <link rel="stylesheet" href="style.css" />
+    <script src="main.js"></script>
```

Comment:
Try adding the 'defer' attribute to this script tag. You might run into an issue if your script is executed before the browser is able to parse the rest of the DOM.